.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '*' pattern is global owners.
+
+# Order is important. The last matching pattern has the most precedence.
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+# Global rule:
+*           @ngworker/ngworkers
+
+# Spectacular:
+/packages/spectacular/**  @LayZeeDK
```

Comment:
I think the right way would be

`/packages/spectacular/ @LayZeeDK ` 

See:

```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com

# In this example, @octocat owns any file in an apps directory
# anywhere in your repository.
apps/ @octocat

# In this example, @doctocat owns any file in the `/docs`
# directory in the root of your repository and any of its
# subdirectories.
/docs/ @doctocat
```

Reply:
I got it from https://github.com/dotnet/samples/blob/master/.github/CODEOWNERS#L18

Reply:
I got it from here https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Reply:
I think they may be the same.

Reply:
`(...)/**` makes more sense to me :-)

Reply:
Me too, but I got confused while reading the docs.

Reply:
Would you add your name to the spectacular Readme and/or package.json?

Reply:
Maybe package.json. A code owner icon should be displayed in GitHub when browsing that part of the workspace.